Converters/CSi/Speckle.Converters.ETABSShared/ToSpeckle/Helpers/EtabsFramePropertiesExtractor.cs

```diff
@@ -77,7 +77,7 @@ private string GetDesignProcedure(CsiFrameWrapper frame)
 
   private string GetSpringAssignmentName(CsiFrameWrapper frame)
   {
-    string springPropertyName = "None"; // Is there a better way to handle null?
+    string springPropertyName = string.Empty;
```

Comment:
Did you want empty string instead of null?  I'm not sure if you were asking someone else or just the API

Reply:
Playing around with what's better for the viewer, filtering etc. More a note to self. Having an initial "None" string just seemed odd.